packages/network/statistics/index.js

```diff
@@ -0,0 +1,18 @@
+import { compose } from 'redux';
+import withData from "@common/utilities/withData";
+import { withTranslation } from "react-i18next";
+import Statistics from './statistics';
+import { getNetworkStatistics } from '@network/utilities/api';
+import { tokenMap } from '@token/configuration/tokens';
+
+export default compose(
+  withData({
```

Comment:
Should Api call not be moved to a separated manager file?

Reply:
Hi, The API call is already in a dedicated file. And the API utility is already moved to the network domain directory. I'm not sure what you are aiming to change?